components/Navigation/Navigation.js

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+import {
+  StyledLink,
+  StyledList,
+  StyledListItem,
+  StyledNavigation,
+} from "./Navigation.styled";
+
+export default function Navigation() {
+  return (
+    <StyledNavigation>
+      <StyledList>
+        <StyledListItem>
+          <StyledLink href="/">Home</StyledLink>
+        </StyledListItem>
+        <StyledListItem>
+          <StyledLink href="/new">Neu</StyledLink>
+        </StyledListItem>
+
+        <StyledListItem>
+          <StyledLink href="/">Archiv</StyledLink>
+        </StyledListItem>
```

Comment:
```suggestion
```
We only want to implement elements that already have a functionality. We never add "empty" buttons or links that lead to nowhere. So please only add this Listitem when you are working on the userstory where you write the logic for the archive :)